cccs-build/azure-pipelines.yml

```diff
@@ -26,7 +27,7 @@ steps:
   - bash: |
       BRANCH_NAME=$(echo "$SYSTEM_PULLREQUEST_SOURCEBRANCH $BUILD_SOURCEBRANCH" | sed -r 's/^\s*(refs\/heads\/)?(\S*).*$/\2/' | sed 's/\//_/g')
       echo "##vso[task.setvariable variable=BRANCH_NAME]$BRANCH_NAME"
-      TRINO_VERSION=$(./mvnw --quiet help:evaluate -Dexpression=project.version -DforceStdout)
+      TRINO_VERSION=${{ variables.trinoVersion }}
```

Comment:
Our buildhost only goes up to Java 21 but this call doesn't work anymore on Java 21. It happened once before and I feel like it's easier to just hardcode the version number instead.